deployment/scripts/mongod/v6.0/26_CRO-181_add_more_roles_for_INGEST_APP_ref.js

```diff
@@ -0,0 +1,22 @@
+print('START 26_CRO-181_add_more_roles_for_INGEST_APP_ref.js');
+
+db = db.getSiblingDB('iam');
+
+db.profiles.updateMany(
+  {
+    applicationName: 'INGEST_APP'
+  },
+  {
+    $addToSet: {
+      roles: {
+        $each : [
+            { name : "ROLE_GET_FILE_FORMATS"},
```

Comment:
Merger ces rôles dans le script précédent, pas la peine d'en avoir 2.

Reply:
Ce sont des devs qui ont été fait sur des itérations différentes -> d'ou le pourquoi il existe deux fichiers et non un pour alimenter les rôles -> cela faisait sens au moment du développement de notre côté mais de votre point de vue il est vrai que ce n'est pas utile. J'ai fais la modification.